packages/cli/openapi-parser/src/__test__/__snapshots__/vellum.test.ts.snap

```diff
@@ -596,25 +588,17 @@ Creates a new document index.",
                   {
                     "key": {
                       "type": "string",
-                      "value": "indexing_config",
+                      "value": "key",
```

Comment:
any thoughts on why the `value` here is now `key` ?

Reply:
Chatted offline, but this was similarly intentional for the same reason as the other comment! And since these are the example TRs, this is what we wanna see